ios/Classes/SwiftHmssdkFlutterPlugin.swift

```diff
@@ -14,17 +14,24 @@ public class SwiftHmssdkFlutterPlugin: NSObject, FlutterPlugin, HMSUpdateListene
     var previewEventChannel: FlutterEventChannel?
     var logsEventChannel: FlutterEventChannel?
     var rtcStatsEventChannel: FlutterEventChannel?
+    var sessionEventChannel: FlutterEventChannel?
 
     var eventSink: FlutterEventSink?
     var previewSink: FlutterEventSink?
     var logsSink: FlutterEventSink?
     var rtcSink: FlutterEventSink?
+    var sessionSink: FlutterEventSink?
+
     var roleChangeRequest: HMSRoleChangeRequest?
 
     internal var hmsSDK: HMSSDK?
 
     private var isStatsActive = false
 
+    internal var sessionStore: HMSSessionStore?
+
+    private var sessionStoreChangeObservers = [HMSSessionStoreKeyChangeListener]()
```

Comment:
@Decoder07 this `sessionStoreChangeObservers` can be used to set single or multiple observers for different or same keys

Reply:
but how can we identify where to give(in which class) the updates.